library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/SectionedRecyclerViewAdapter.java

```diff
@@ -290,7 +320,12 @@ public int getItemViewType(int position) {
 
                 switch (section.getState()) {
                     case LOADED:
-                        return viewType + 2;
+                        int itemViewType = section.getItemViewType(getPositionInSection(position));
+                        if (itemViewType == -1) {
+                            return viewType + 2;
+                        } else {
+                            return itemViewType;
```

Comment:
[DESIGN] The default method getItemViewType returns -1 unless it is Overrriden. So if the user chooses not to use CustomViewTypes, it returns -1 and defaults to the original logic. Otherwise, if not -1, getItemViewType returned a CustomViewType and that should be returned.